tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature

```diff
@@ -27,20 +27,20 @@ Feature: restrict Sharing
     And user "Brian" has uploaded file "lorem.txt" to "simple-folder/lorem.txt" in the server
     And user "Brian" has logged in using the webUI
 
-  @smokeTest
+  @smokeTest @skipOnOCIS
   Scenario: Restrict users to only share with users in their groups
+#   shares-only-with-groups-members configuration is not available now in the oCIS webUI
```

Comment:
```suggestion
#   shares-only-with-groups-members configuration is not available yet in the oCIS webUI
```

Reply:
Maybe issue tag of this https://github.com/owncloud/ocis/issues/1328 issue will also be relevant here?

Reply:
ISSUE TAG 1317 is present at the top and 1317 and 1328 are inter-related. I think this will overload issue tag

Reply:
Just because they are interrelated doesn't mean they are the same thing. If issue tag 1317 is at the top you can add 1328 alongside it. I'm just asking this for documentation as in issue https://github.com/owncloud/ocis/issues/1328#issuecomment-1184237905 it's mentioned here that it'll be implemented in ocis spaces too